### DISCUSSION TITLE
Can I build a single glossary from several glossary.md input files

### DISCUSSION THREAD

**Discussion by TomLav** [executablebooks/meta#870] [Q&A]
Hello,

We are building a series of technical JupyterBooks that share some terms / acronyms. Each book might have its own acronyms in addition to the set of common acronyms.

Today we have to duplicate copy/paste the terms that are common in each glossary.md file (they also contain the terms that are specific to the book). This is a manual task.

Ideally, there would be  a way to build a single glossary from several .md files. Then the books would share a common common_glossary.md file, and have in addition a my_glossary.md file. The resulting glossary section would be a mix of the two.

Is there a way to do that? Maybe using 'include' files (if that exists in jupyterbooks). Thanks
Thomas

**Comment by rowanc1** [comment]
Starting to think about this in the `mystmd` project:
https://github.com/executablebooks/mystmd/issues/1099
Thanks for the pointers over in twitter/x as well!
